.github/dependabot.yaml

```diff
@@ -1,27 +1,21 @@
+---
 version: 2
 updates:
-- package-ecosystem: cargo
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 25
-  commit-message:
-    prefix: "chore(deps)"
+  - package-ecosystem: cargo
+    directory: "/"
+    schedule:
+      interval: daily
+    commit-message:
+      prefix: "chore(deps)"
 
-  # See https://github.com/dependabot/dependabot-core/issues/2824 for doing
-  # multiple directories for the same check.
-- package-ecosystem: cargo
-  directory: "/panic/abort"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 25
-  commit-message:
-    prefix: "chore(deps)"
-
-- package-ecosystem: github-actions
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
-  commit-message:
-    prefix: "chore(deps)"
+  - package-ecosystem: github-actions
```

Comment:
Note the panic/abort no longer exists it just failed to get cleaned up when originally refactored